src/ExceptionConverterHelper.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App;
+
+use Keboola\DockerBundle\Docker\Runner\Output;
+use Keboola\DockerBundle\Exception\UserException;
+use Keboola\ErrorControl\Message\ExceptionTransformer;
+use Keboola\JobQueueInternalClient\JobFactory\JobResult;
+use Keboola\ObjectEncryptor\Exception\UserException as EncryptionUserException;
+use Psr\Log\LoggerInterface;
+use Throwable;
+
+class ExceptionConverterHelper
+{
+    public static function convertExceptionToResult(
+        LoggerInterface $logger,
+        Throwable $e,
+        string $jobId,
+        array $outputs
+    ): JobResult {
+        $errorType = is_a($e, UserException::class) || is_a($e, EncryptionUserException::class)
```

Comment:
Jen tak ze zvedavosti - nejaky konkretni preference pro `is_a` misto `instanceof`? (krom toho, ze je to kratsi :))

Reply:
ne, asi jen to, že je to kratší :)